### ISSUE TITLE
[Question] Understanding Train/Test Outputs

### ISSUE THREAD

**Issue by ericy51** [tensorflow/recommenders#652]
Hello,

I was trying to understand the training outputs from the two tower retrieval model I modified by initializing the item tower with my own embeddings. Initially the top k metrics look pretty good but after the 3rd epoch there's a massive drop off. Could someone help me understand how to interpret the metrics after the 3rd epoch. Also why might we see this kind of behavior?

![image](https://user-images.githubusercontent.com/16570133/223566460-6ac9ecd5-3e98-4df5-aa01-5cce7b9b1521.png)

**Comment by patrickorlando** [comment]
Most likely numerical instability. Tune your learning rate. Apply L1 or L2 weight normalisation.
